mmdeploy/utils/config_utils.py

```diff
@@ -393,3 +393,51 @@ def get_dynamic_axes(
                 raise KeyError('No names were found to define dynamic axes.')
         dynamic_axes = dict(zip(axes_names, dynamic_axes))
     return dynamic_axes
+
+
+def get_normalization(model_cfg: Union[str, mmcv.Config]):
+    model_cfg = load_config(model_cfg)[0]
+    pipelines = model_cfg.data.test.pipeline
+    for i, pipeline in enumerate(pipelines):
+        if pipeline['type'] == 'MultiScaleFlipAug':
+            assert 'transforms' in pipeline
+            for trans in pipeline['transforms']:
+                if trans['type'] == 'Normalize':
+                    return trans
+        else:
+            if pipeline['type'] == 'Normalize':
+                return pipeline
+
+
+def get_rknn_quantization(deploy_cfg: mmcv.Config):
+    """Get the flag of `do_quantization` for rknn backend.
+
+    Args:
+        deploy_cfg (mmcv.Config): The content of config.
+
+    Returns:
+        bool: Do quantization or not.
+    """
+    if get_backend(deploy_cfg) == Backend.RKNN:
```

Comment:
So we did not reuse calib_config?

Reply:
Yes, quantization is controlled by a flag in RKNN.

Reply:
How did you feed dataset?

Reply:
by a txt file containing paths of images.